desktop/src/Desktop/Frontend.hs

```diff
@@ -123,18 +128,28 @@ bipWallet
   => AppCfg Crypto.XPrv t (RoutedT t (R FrontendRoute) (BIPCryptoT m))
   -> RoutedT t (R FrontendRoute) m ()
 bipWallet appCfg = do
-  mRoot <- getItemStorage localStorage BIPStorage_RootKey
-  rec
-    root <- holdDyn (mRoot, Nothing) upd
-    upd <- switchHold never <=< dyn $ ffor root $ \case
-      (Nothing, _) -> do
-        keyAndPass <- runSetup
-        performEvent $ ffor keyAndPass $ \(x, Password p) -> do
+  let
+    runSetup0 r fromLockScreen = do
```

Comment:
0 ?

Reply:
It's either that or `runSetup'`, I just needed a name.